schemas/blocks/section_summary.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "block": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "../common_definitions.json#/id"
+      },
+      "type": {
+        "type": "string",
+        "enum": ["Section-Summary"]
```

Comment:
I think the pattern for the enum is CamelCase with no hyphen
so this should be `SectionSummary`